src/ucp/wireup/select.c

```diff
@@ -627,10 +679,11 @@ ucp_wireup_add_memaccess_lanes(ucp_wireup_select_ctx_t *select_ctx,
 
     /* Add to the list of lanes and remove all occurrences of the remote md
      * from the address list, to avoid selecting the same remote md again. */
-    ucp_wireup_add_lane_desc(select_ctx, &select_info, dst_md_index, usage, 0);
+    ucp_wireup_add_lane(select_ctx, &select_info, usage);
+
     remote_md_map &= ~UCS_BIT(dst_md_index);
-    tl_bitmap = ucp_wireup_unset_tl_by_md(ep, tl_bitmap,
-                                          select_info.rsc_index);
+    tl_bitmap      = ucp_wireup_unset_tl_by_md(ep, tl_bitmap,
```

Comment:
maybe pass &tl_bitmap, &remote_md_map, and &select_info to ucp_wireup_unset_tl_by_md?
so it will handle both local and remote bitmaps
to unite common code with 712-717

Reply:
done